File: unittest/scripts/auto/py_devapi/scripts/mysqlx_collectionremove_help.py
shell.connect(__uripwd)
schema = session.create_schema('collectionremove_help')
coll = schema.create_collection('sample')
collremove = coll.remove('1=1');
session.close()

#@ collremove
collremove.help()

#@ global ? for CollectionRemove[USE:collremove]
\? CollectionRemove

#@ global help for CollectionRemove[USE:collremove]
\help CollectionRemove

#@ collremove.bind
collremove.help('bind')

#@ global ? for bind[USE:collremove.bind]
\? CollectionRemove.bind

#@ global help for bind[USE:collremove.bind]
\help CollectionRemove.bind

#@ collremove.execute
collremove.help('execute')

#@ global ? for execute[USE:collremove.execute]
\? CollectionRemove.execute

#@ global help for execute[USE:collremove.execute]
\help CollectionRemove.execute

#@ collremove.help
collremove.help('help')

#@ global ? for help[USE:collremove.help]
\? CollectionRemove.help

#@ global help for help[USE:collremove.help]
\help CollectionRemove.help

#@ collremove.limit
collremove.help('limit')

#@ global ? for limit[USE:collremove.limit]
\? CollectionRemove.limit

#@ global help for limit[USE:collremove.limit]
\help CollectionRemove.limit

#@ collremove.remove
collremove.help('remove')

#@ global ? for remove[USE:collremove.remove]
\? CollectionRemove.remove

#@ global help for remove[USE:collremove.remove]
\help CollectionRemove.remove

#@ collremove.sort
collremove.help('sort')

#@ global ? for sort[USE:collremove.sort]
\? CollectionRemove.sort

#@ global help for sort[USE:collremove.sort]
\help CollectionRemove.sort

shell.connect(__uripwd)
session.drop_schema('collectionremove_help');
session.close()

File: unittest/scripts/auto/js_devapi/scripts/mysqlx_collectionremove_help.js
//@ Initialization
shell.connect(__uripwd);
var schema = session.createSchema('collectionremove_help');
var collection = schema.createCollection('sample');
var crud = collection.remove("true")
session.close();

//@ CollectionRemove help
crud.help();

//@ CollectionRemove help, \? [USE:CollectionRemove help]
\? CollectionRemove

//@ Help on bind
crud.help('bind');

//@ Help on bind, \? [USE:Help on bind]
\? CollectionRemove.bind

//@ Help on execute
crud.help('execute');

//@ Help on execute, \? [USE:Help on execute]
\? CollectionRemove.execute

//@ Help on help
crud.help('help');

//@ Help on help, \? [USE:Help on help]
\? CollectionRemove.help

//@ Help on limit
crud.help('limit');

//@ Help on limit, \? [USE:Help on limit]
\? CollectionRemove.limit

//@ Help on remove
crud.help('remove');

//@ Help on remove, \? [USE:Help on remove]
\? CollectionRemove.remove

//@ Help on sort
crud.help('sort');

//@ Help on sort, \? [USE:Help on sort]
\? CollectionRemove.sort

//@ Finalization
shell.connect(__uripwd);
session.dropSchema('collectionremove_help');
session.close();
